### PR TITLE
Add K8s v1.19 test job to circleci

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

**Review by keitwb — APPROVED** [review]

**Comment by keitwb** [comment]
Looks like the tests needs to be fixed up.

**Comment by jchengsfx** [comment]
> Looks like the tests needs to be fixed up.

I think the tests are failing for v1.19 due to api changes (https://kubernetes.io/docs/setup/release/notes/), but will need someone to confirm.

**Comment by keitwb** [comment]
@jchengsfx I don't see anything in the release notes that explains it.  What are you seeing?  My guess would be something wrong with the minikube setup.

**Comment by jchengsfx** [comment]
> @jchengsfx I don't see anything in the release notes that explains it. What are you seeing? My guess would be something wrong with the minikube setup.

The `kublet-metrics` and `kubernetes-volumes` monitor tests are failing (not getting any datapoints) for v1.19 (the tests pass for the other versions).  https://app.circleci.com/pipelines/github/signalfx/signalfx-agent/8378/workflows/2f306b04-0a16-44cb-b9b2-c8a220ec2607/jobs/238100/steps

I don't see any minikube options specific to v1.19 or anything in the agent logs, but will check around some more.

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   machine_image:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-1604:202007-01
```
Comment (jchengsfx) [inline .circleci/config.yml:6]: @keitwb I think the test failures are related to https://github.com/kubernetes/kubernetes/issues/94281.  Updating to the latest machine image (from docker 18.09 to 19.03) resolves the failures.

Comment (keitwb) [reply .circleci/config.yml:6]: Looks like the metrics

```
blkio.io_service_bytes_recursive.read',
E         -            'blkio.io_service_bytes_recursive.write',
```
Are missing in the new docker version.  Just make the tests that look for them ignore them by removing them from the asserted set in pytest.

**Pull request merged**